### ISSUE TITLE
Related to Issue #662 

### ISSUE THREAD

**Issue by G4OEU** [DistributedProofreaders/guiguts#669]
**Describe the bug**
The 1.3 Manual says:

> Note:The opening and closing markers (/*, etc.) should stand alone on a line, but these markups must be preceded and followed by blank lines. If the blank lines are missing, Guiguts will fail to recognize either the beginning or the end of the markup and mis-wrapping will result.

so you add a blank line to keep the markers separated. This is needed, for example, after page separators have been removed. 

At the end of the same section of the 1.3 Manual it says:

> Clean Up Rewrap Markers

> Removes the Block Markers (such as '/* */ /# #/ /$ $/ /p p/') from the document, and removes extra blank lines that may have been needed to keep them separated from each other for rewrapping. Once you've done this and saved the result, it'll be more difficult to do some kinds of searches and automatic rewrapping, so this usually is one of the very last steps in preparing the Plain Text version of the book.

However **Clean Up Rewrap Markers** does not remove the extra blank line that was added to keep the rewrap markers separated as shown below:
```
...
*/

/*
...
```

**To Reproduce**
1. Open Guiguts with [bug-clean-up-rewrap-markers.txt](https://github.com/DistributedProofreaders/guiguts/files/6232749/bug-clean-up-rewrap-markers.txt)
2. Tools->Rewrap All
3. Tools->Clean Up Rewrap Markers
4.  The resulting text should have the same spacing between lines but you will observe that there is an extra line that the cleanup failed to remove.

**Expected behavior**
Tools->Clean Up Rewrap Markers should removes extra blank lines that may have been needed to keep rewrap markers separated

**Environment**
* Guiguts version: 1.3-beta-test2
* OS and version: 
* Perl version: 
* Perl distribution: 

**Screenshots**

**Additional context**

**Comment by windymilla** [comment]
My understanding (and it's mentioned [here](https://www.pgdp.net/wiki/DP_Official_Documentation:PP_and_PPV/Post-Processing_FAQ#Do_a_first-pass_check) in the PP FAQ) is that early on, it's best to remove cross-page markup where you actually want the table/poem to join over the page boundary. So strictly, in your previous example, you should remove entirely the
'''
$/
/$
'''

The problem with this request is that it isn't possible for the software to know whether you wanted two separate tables/poems, separated by a blank line or whether it was intended to be all one table/poem. So, Clean Up Rewrap Markers has to leave the blank line there.

In summary, if it's a long table/poem that crosses pages, remove the adjacent close/open markup. If it's two tables, make sure there is a blank line between them.

If this seems reasonable, you can close this, or I can - thanks

**Comment by charliehoward4dp** [comment]
The 1.3 Manual's documentation has been updated by G4OEU and me, so this issue can be closed.

**Comment by windymilla** [comment]
Closing as per above comment
